pkg/conference/config.go

```diff
@@ -2,7 +2,11 @@ package conference
 
 // Configuration for the group conferences (calls).
 type Config struct {
-	// Keep-alive timeout for WebRTC connections. If no keep-alive has been received
-	// from the client for this duration, the connection is considered dead (in seconds).
+	// Keep-alive timeout for WebRTC connections. If the client doesn't respond
+	// to an `m.call.ping` with an `m.call.pong` for this amount of time, the
+	// connection is considered dead. (in seconds, no greater then 30)
 	KeepAliveTimeout int `yaml:"timeout"`
+	// The time after which we should send another m.call.ping event to the
+	// client. (in seconds, greater then 30)
+	PingInterval int `yaml:"pingInterval"`
```

Comment:
NB: If we're already extending the configuration for the keepalive/heartbeat logic, would it make sense to group them together in their own struct: `KeepAliveConfig` ([though keep-alive is possibly not the best term here](https://github.com/matrix-org/matrix-spec-proposals/pull/3898#discussion_r1042049466)) or `HeartBeatConfig`? So that it's obvious that both fields actually go together (it would also mean, we could supply this structure with grouped/related fields directly to the one who needs it instead of passing fields one-by-one [passing one-by-one can mix the order due to typos etc and also might be less obvious])

Reply:
Have added it to the list of config stuff to fix in a follow up PR: https://github.com/matrix-org/waterfall/issues/71